src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -35,6 +37,7 @@ public TemplateAnalyzer(string template, string parameters = null, string templa
             this.Template = template ?? throw new ArgumentNullException(paramName: nameof(template));
             this.Parameters = parameters;
             this.TemplateFilePath = templateFilePath;
+            this.IsBicep = templateFilePath != null && templateFilePath.ToLower().EndsWith(".bicep");
```

Comment:
Add a test for the updates made to this constructor.

Reply:
_Discussed on teams_: don't need these tests for now.